src/vi/goctruyentranhvui/build.gradle

```
@@ -1,8 +1,7 @@
 ext {
     extName = 'Goc Truyen Tranh Vui'
     extClass = '.GocTruyenTranhVui'
-    extVersionCode = 5
-    isNsfw = true
+    extVersionCode = 6
```

Should be explicit to convey that you have checked it.
```suggestion
    extVersionCode = 6
    isNsfw = false
```